kytos/core/switch.py

```diff
@@ -107,6 +113,30 @@ def id(self):  # pylint: disable=invalid-name
         """
         return "{}".format(self.dpid)
 
+    @property
+    def status(self):
+        """Return the status of switch instance.
+
+        Returns:
+            enum: value of SwitchStatus to indicate switch status.
+
+        """
+        if self.enabled and self.active:
+            return SwitchStatus.UP
+        elif self.is_administration_down():
+            return SwitchStatus.ADMINISTRATIVE_DOWN
+        else:
+            return SwitchStatus.OPERATION_DOWN
+
+    def is_administration_down(self):
+        """Verify if the switch is administration down.
+
+        Returns:
+            boolean: True if the switch is administration down, otherwise False
```

Comment:
On the top you use "administratIVE", here it is "administratION" three times. It should be "administrative" everywhere, right?

Reply:
On another note: is this method really necessary if all it does is return "not enabled"?

Reply:
I guess this method is necessary to be used in another methods. I am only implementing the blueprint specification, this method is defined in: https://github.com/kytos/kytos/blob/master/docs/blueprints/EP013.rst

Reply:
I am wrong with the "administratIVE" and "administratION", i will change the words to "administratIVE".